backend/dataall/aws/handlers/sagemaker_studio.py

```diff
@@ -7,17 +7,18 @@
 
 class SagemakerStudio:
     @staticmethod
-    def client(AwsAccountId, region):
-        session = SessionHelper.remote_session(AwsAccountId)
+    def client(AwsAccountId, region, role=None):
+        session = SessionHelper.remote_session(accountid=AwsAccountId, role=role)
         return session.client('sagemaker', region_name=region)
 
     @staticmethod
-    def get_sagemaker_studio_domain(AwsAccountId, region):
+    def get_sagemaker_studio_domain(AwsAccountId, region, role):
```

Comment:
We should add a role to `resolvers` of `SagemakerStudio`. Otherwise, we will crash when we try to create a profile

Reply:
You are right, I fixed it by adding a default for the parameter and letting the pivotRole be the used role